content/StsServerIdentity/Services/Certificate/CertificateService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Security.Cryptography.X509Certificates;
+
+namespace StsServerIdentity.Services.Certificate
+{
+    public static class CertificateService
+    {
+        public static (X509Certificate2, X509Certificate2) GetCertificates(CertificateConfiguration certificateConfiguration)
+        {
+            (X509Certificate2, X509Certificate2) certs = (null, null);

```

Comment:
You could use a named tuple which would make the code more readable. Then you would not use `Item1` but instead `Foo`
```suggestion
            (X509Certificate2 Foo, X509Certificate2 Bar) certs = (null, null);
```

Reply:
thanks, didn't know this :) fixed